meetings/2025-10-02.md

```diff
@@ -0,0 +1,147 @@
+- 2025-10-02
+
+## Details
+
+- call https://meet.jit.si/solid-practitioners
+- chat https://matrix.to/#/#solid-practitioners:matrix.org
+- repo https://github.com/solid-contrib/practitioners
+- recordings https://spectra.video/c/solid_practitioners/videos
+
+## Facilitator
+
+- Michal
+
+## Scribe
+
+- Theo
+
+## Participants (please add yourself)
+
+- Jeff Zucker
+- [Luke Dary](https://www.linkedin.com/in/lukedary)
+- [Michal](https://id.mrkvon.org)
+- Tom Byrd
+- [Fred Gibson](https://graphmetrix.com)
+- [Theo @thhck](http://github.com/thhck)
+- [Rui Zhao](https://me.ryey.icu)
+- Marc Haddle
+- [Jesse Wright](https://jeswr.solidcommunity.net/)
+- [Wilfred Pinfold](https://www.linkedin.com/in/wilfred-pinfold-aa6b68/)
+- John Kirkwood
+- \_
+
+## Agenda
+
+### Check-in
+
+- How are you coming to the meeting? (brief, round, 3-5 min, not minuted)
+
+### Administrative
+
+- Call for Scribe
+- ~~Objections to recording session?~~ (not recorded)
+- Review agenda
+
+### New Member Check-in (brief intro, current work)
+
+- Luke: work at Redhat. Here to talk about adoption of Solid. Also volunteer with non-profit, which pay money for auth or storage, would be a lower cost for them to use Solid.
+- ?: You talk also about issue convincing ppl at Redhat to use Solid ?
+- L: There is a need to capture user data ( lightmode, language .. ), because of our ecosystem at Redhat and try to bring them together using GraphQL and I proposed Solid at a solution and the answer was "we already have system that can do what Solid can do ( auth etc...)" . I try to convice them that Solid is a better option. In there perspective if user don't request it, no need to implement it.
+- Jesse: At the ODI we are working on education and training project. How do you convice SME and org to use Solid. For SME, a good argument in favor of using Solid is they don't have to worry about the compliance issue . Would like to work with Luke to work on this issue ( convincing to use Solid)
+- WP : With business, is more simple if they don't have to manage the data. [...]
+- WP: In the economics of decentralised data and trading data the wallet is a great solution for that.
+- Luke: A big issue is that ontology internally is all over the place, ppl have hard time to believe they can be one unique source of truth coming from the user
+- WP: will be great if redhat provide some infrastructure, as it is currently very embrionic
+- JZ: Another important part of Solid is interoperability
+- Tom Byrd: Another good selling point: [...]
+- Marc Haddle: When it come to patient data: way better if have it in one place. Can be challenging to maintain the link of user data ( because of de-auth ). As long as data is hosted on someone else's server, you don't have control.
+- Luke: Can be interesting for user centric AI-agent . ( Ai is currently a good selling point)
+- WP: AI has been a good selling point in our side. If redhat really want to differentiate itself from others, this can be a good point.
```

Comment:
```suggestion
- WP: AI has been a good selling point on our side. If RedHat really wants to differentiate itself from others, this can be a good point.
```